namerd/core/src/main/scala/io/buoyant/namerd/DtabSerialization.scala

```diff
@@ -0,0 +1,37 @@
+package io.buoyant.namerd
+
+import com.fasterxml.jackson.core.{JsonGenerator, JsonParser}
+import com.fasterxml.jackson.databind.{DeserializationContext, JsonDeserializer, JsonSerializer, SerializerProvider}
+import com.fasterxml.jackson.databind.module.SimpleModule
+import com.twitter.finagle.{NameTree, Path}
+
+object DtabSerialization {
```

Comment:
Hmm, there's already a `DtabCodec` declared in HttpControlService that does something different. Are you okay with the name duplication?